models/note.go

```diff
@@ -22,7 +23,19 @@ var categoryStrings = [...]string{
 	"predictions",
 }
 
+var CannotDeserializeCategoryStringError = errors.New("String does not correspond to a Note Category")
+
+func DeserializeCategory(input string) (Category, error) {
+	for i := 0; i < len(categoryStrings); i++ {
+		if input == categoryStrings[i] {
+			return Category(i), nil
```

Comment:
We'll just remember to think about this more later

Reply:
in regards to what? i can add a comment if you'd like

Reply:
sorry, i'm referring to an earlier conversation we had about considering whether or not to just pass around int values for this enum to the front end. We can just deal with it later.